### PR TITLE
testing.

### PR DESCRIPTION
**Pull request by EbenZhang**

### CODE REVIEW

**Comment by EbenZhangEmbed** [comment]
###### *PReviewer.GitExt/GitUIPluginInterfaces.dll*

testat

###### *PReviewer.GitExt/PReviewer.ext.cs*

asetwaerfasdf

###### *PReviewer.GitExt/Properties/AssemblyInfo.cs*

* aslkdjflaskdjf
al,sdkfjal;sdkjf

* ;lasjdflakjsdf

###### *PReviewer.Test/TestLoginVm.cs*

* aslkdjfsdfasdflaskdjf
al,sdkfjal;sdkjf
asdf
* ;lasjdflakjsdf

###### *General Comments*

* adfasesafdasdf
* lasjdflkasdf
asd;lkjfa;lskdf
asdl;kjf

a;sldkjf;lasjdf;lkasjdf

File: PReviewer.GitExt/PReviewer.ext.cs [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+            get { return "PReviewer"; }
+        }
+
+        public override bool Execute(GitUIBaseEventArgs gitUiCommands)
+        {
+            var userProfile = Environment.GetFolderPath(Environment.SpecialFolder.Programs);
+            var path = Path.Combine(userProfile, @"Nicologies\PReviewer.appref-ms");
+            if (File.Exists(path))
+            {
+                Process.Start(path);
```
Comment (EbenZhang) [inline PReviewer.GitExt/PReviewer.ext.cs]: test

File: LICENSE.md [hunk 2]
```diff
@@ -8,12 +8,12 @@ All rights reserved [Nicologies](http://www.nicologies.tk) @2015
...
 If you feel that the work I provide is worth it, you can donate by using the Donate button below. This is greatly appreciated.
 You can use Paypal to donate even if you do not have a Paypal account.There is, of course, no obligation whatsoever. 
-If you prefer, you can rather get me a bottle of coke next time I am in a location near you ;) 
+If you prefer, you can rather buy me a bottle of coke next time I am in a location near you ;) 
 
 <form action="https://www.paypal.com/cgi-bin/webscr" method="post" target="_top">
 <input type="hidden" name="cmd" value="_donations">
 <input type="hidden" name="business" value="nicoleflopy@gmail.com">
-<input type="hidden" name="lc" value="US">
+<input type="hidden" name="lc" value="AUS">
```
Comment (EbenZhang) [inline LICENSE.md]: test

File: .gitmodules [hunk 3]
```diff
@@ -1,6 +1,6 @@
 [submodule "WpfCommon"]
 	path = WpfCommon
-	url = git@github.com:EbenZhang/WpfCommon.git
```
Comment (EbenZhang) [inline .gitmodules]: test

**Pull request closed without merging**